modules/openapi-generator/src/main/resources/swift5/api.mustache

```diff
@@ -167,7 +167,7 @@ extension {{projectName}}API {
                 switch result {
                 {{#returnType}}
                 case let .success(response):
-                    promise(.success(response.body!))
+                    promise(.success(response.body))
```

Comment:
@funzin please search and replace all occurrences of `response.body!` with `response.body` in this class, to fix all the cases. Thanks

Reply:
I have already checked `response.body!` in `swift5`
dbdc585

Maybe, CI will be succeed.

Reply:
@4brunu 
CI ✅